tachyon/math/finite_fields/small_prime_field.h

```diff
@@ -24,8 +24,7 @@ namespace tachyon::math {
 
 // A prime field is finite field GF(p) where p is a prime number.
 template <typename _Config>
-class PrimeField<_Config, std::enable_if_t<!_Config::kIsSpecialPrime &&
-                                           (_Config::kModulusBits <= 32) &&
+class PrimeField<_Config, std::enable_if_t<(_Config::kModulusBits <= 32) &&
```

Comment:
Was the previous `!_Config::kIsSpecialPrime` unneeded here?

Reply:
Yes, I think the flag `kIsSpecialPrime` is changed to `kUseAsm` in this PR. Since there's no assembly code for a Small Prime Field, it doesn't need to have this condition any more.